products/privatebin.md

```diff
@@ -0,0 +1,68 @@
+---
+title:  PrivateBin
+category: server-app
+tags: pastebin encryption privacy-security
+permalink: /privatebin
+alternate_urls:
+-   /private-bin
```

Comment:
```suggestion
```

PrivateBin is always referred as `PrivateBin`, I did not found any reference to `Private Bin`.